processing/src/main/java/io/druid/query/topn/DimValHolder.java

```diff
@@ -26,13 +26,13 @@
 public class DimValHolder
 {
   private final Object topNMetricVal;
-  private final String dimName;
+  private final Comparable dimName;
```

Comment:
It does, I think "dimValue" would probably be a better name but didn't feel like renaming it in this PR